theme_enark/views/snippets/s_cover.xml

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-
-<template id="s_cover" inherit_id="website.s_cover">
```

Comment:
Let's not remove templates for now unless really necessary.
We'll have to review each theme, chances are that we'll have to re-create the very same template but applying other customizations ;)
Replace the xpath with something that makes sense for this theme... adjust the padding maybe?